code/src/App.js

```diff
@@ -1,12 +1,27 @@
 import React from 'react';
 import data from './data.json';
-
-console.log(data);
+import Header from './components/Header';
+import Albums from './components/Albums';
 
 export const App = () => {
-  return (
-    <div>
-      Find me in src/app.js!
-    </div>
-  );
+    return (
+        <>
+            <section className = 'outer-wrapper'>
+
+                <div className = 'inner-wrapper'>
+
+                    <Header />
+
+                    <section className='album-container'>
+                            {data.albums.items.map(item => {
+                                return <Albums key={item.id} data={item}/>
+                            })}
+                    </section>
+
+                </div>
+
+            </section>
+
+        </>
+    );
```

Comment:
Clean and clear code nice work on keeping the app code to a minimum and storing most of the information in the components.